packages/menu/src/Menu.tsx

```diff
@@ -12,14 +12,33 @@ const rootMenuStyle = css`
   background-color: ${colors.mongodb.white};
 `;
 
-const EscapeKey = 27;
-
-interface MenuProps extends Omit<PopoverProps, 'spacing'> {
+interface MenuProps extends Omit<PopoverProps, 'spacing, active'> {
```

Comment:
```suggestion
interface MenuProps extends Omit<PopoverProps, 'spacing' | 'active'> {
```
I had to look this up heh, https://stackoverflow.com/questions/48215950/exclude-property-from-type

Reply:
👍